diff --git a/common/Android.bp b/common/Android.bp
index dd6d40c..3fbe855 100644
--- a/common/Android.bp
+++ b/common/Android.bp
@@ -1066,7 +1066,6 @@ android_app_import {
 	name: "GoogleDialer",
 	owner: "gms",
 	apk: "proprietary/product/priv-app/GoogleDialer/GoogleDialer.apk",
-	overrides: ["Dialer"],
 	preprocessed: true,
 	presigned: true,
 	dex_preopt: {
@@ -1080,7 +1079,6 @@ android_app_import {
 	name: "PrebuiltBugle",
 	owner: "gms",
 	apk: "proprietary/product/priv-app/PrebuiltBugle/PrebuiltBugle.apk",
-	overrides: ["messaging"],
 	preprocessed: true,
 	presigned: true,
 	dex_preopt: {
